serverlessworkflow/modules/ROOT/pages/cloud/operator/configuring-workflows.adoc

```diff
@@ -8,7 +8,13 @@ This document describes how to configure a Workflow service with the {operator_n
 
 == Editing the Workflow Configuration
 
-When the operator deploys the Workflow service, it also creates a `ConfigMap` named after the `SonataFlow` object with the suffix `-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props`.
+When the operator deploys the Workflow service, it also creates two `ConfigMap`s to store the runtime properties:
+
+* The user properties are defined in a `ConfigMap` named after the `SonataFlow` object with the suffix `-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props`.
+* The xref:#managed-props[Managed Properties] are defined in a `ConfigMap` named after the `SonataFlow` object with the suffix `-managed-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props`.
```

Comment:
```suggestion
* The xref:#managed-props[Managed Properties] are defined in a `ConfigMap` named after the `SonataFlow` object with the suffix `-managed-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props-managed`.

? I don't remember the name exactly but those names differ, right?
```

Reply:
`service-props` and `service-managed-props` is what I see on my deployed example

Reply:
The generated name is  service-managed-props

Reply:
Updating it, thanks